services/discovery/Makefile

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
```

Comment:
Again ... kind of a duplicate Makefile for now ... this simply sets up the virtualenv and wraps `sam ...` commands for now.